balancer/rls/picker_test.go

```diff
@@ -143,8 +143,8 @@ func (s) TestPick_DataCacheMiss_PendingEntryExists(t *testing.T) {
 		},
 	}
 
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
+	for _, tst := range tests {
```

Comment:
No need to rename `test` to `tst` now that the testing package is called `rlstest`.